docs/SupportedResources/2021-08-01.md

```diff
@@ -0,0 +1,82 @@
+| API Management Operations | - | | |
+| API Management Service | - | | |
+| API Management Service Skus | - | | |
+| Api Operation | ✓ | [1.0.0-beta.2](https://github.com/Azure/azure-api-management-devops-resource-kit/releases/tag/1.0.0-beta.2) | |
+| Api Operation Policy | ✓ | [1.0.0-beta.2](https://github.com/Azure/azure-api-management-devops-resource-kit/releases/tag/1.0.0-beta.2) | |
+| Api Policy | ✓ | [1.0.0-beta.2](https://github.com/Azure/azure-api-management-devops-resource-kit/releases/tag/1.0.0-beta.2) | |
+| Api Product | ✓ | [1.0.0-beta.2](https://github.com/Azure/azure-api-management-devops-resource-kit/releases/tag/1.0.0-beta.2) | |
+| Api Release | ✗ | | |
+| Api Revision | ✓ | [1.0.0-beta.2](https://github.com/Azure/azure-api-management-devops-resource-kit/releases/tag/1.0.0-beta.2) | Probably needs refactoring (Move out of `ApiExtractor`) |
+| Api Schema | ✓ | [1.0.0-beta.2](https://github.com/Azure/azure-api-management-devops-resource-kit/releases/tag/1.0.0-beta.2) | |
+| Api Tags | ✓ | [1.0.0-beta.2](https://github.com/Azure/azure-api-management-devops-resource-kit/releases/tag/1.0.0-beta.2) | |
+| Api Version Set | ✓ | [1.0.0-beta.2](https://github.com/Azure/azure-api-management-devops-resource-kit/releases/tag/1.0.0-beta.2) | probably needs refactoring (Move out of `ApiExtractor`) |
+| APIs | ✓ | [1.0.0-beta.2](https://github.com/Azure/azure-api-management-devops-resource-kit/releases/tag/1.0.0-beta.2) | |
+| Authorization Server | ✓ | [1.0.0-beta.2](https://github.com/Azure/azure-api-management-devops-resource-kit/releases/tag/1.0.0-beta.2) | |
+| Backend | ✓ | [1.0.0-beta.2](https://github.com/Azure/azure-api-management-devops-resource-kit/releases/tag/1.0.0-beta.2) | |
+| Cache | ✗ | | |
+| Certificate | ✗ | | |
+| Content Item | ✗ | | |
+| Content Type | ✗ | | |
+| Delegation Settings | ✗ | | |
+| Deleted Services | - | | |
+| Diagnostic | ✓ | [1.0.0-beta.2](https://github.com/Azure/azure-api-management-devops-resource-kit/releases/tag/1.0.0-beta.2) | Move out of `ApiExtractor` |
+| Email Template | ✗ | | |
+| Gateway | ✓ | [1.0.0-beta.2](https://github.com/Azure/azure-api-management-devops-resource-kit/releases/tag/1.0.0-beta.2) | |
+| Gateway Api | ✓ | [1.0.0-beta.2](https://github.com/Azure/azure-api-management-devops-resource-kit/releases/tag/1.0.0-beta.2) | |
+| Gateway Certificate Authority | ✗ | | |
+| Gateway Hostname Configuration | ✗ | | |
+| Group | ✓ | [1.0.0-beta.2](https://github.com/Azure/azure-api-management-devops-resource-kit/releases/tag/1.0.0-beta.2) | |
+| Group User | ✗ | | |
+| Identity Provider | ✗ | | |
+| Issue | ✗ | | |
+| Logger | ✓ | [1.0.0-beta.2](https://github.com/Azure/azure-api-management-devops-resource-kit/releases/tag/1.0.0-beta.2) | |
+| Network Status | ✗ | | |
+| Notification | ✗ | | |
+| Notification Recipient Email | ✗ | | |
+| Notification Recipient User | ✗ | | |
+| Notification Recipient User | ✗ | | |
+| OpenId Connect Provider | ✗ | | |
+| Operation | - | | |
```

Comment:
We want to extract a set of APIs by tag, and I think this is the appropriate call.

Reply:
There is a [API Tag Description](https://docs.microsoft.com/en-us/rest/api/apimanagement/current-ga/api-tag-description) operation group, isn't it what you are talking about? Because in [Operation](https://docs.microsoft.com/en-us/rest/api/apimanagement/current-ga/operation) there is only `list by tags` call, I dont think it helps us with any resources, isn't it?

Reply:
Sorry, I got confused. Marked `Operation` as not supported for now